python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -249,6 +260,7 @@ def solid(self) -> Node:
         """
         return self._step_execution_context.pipeline_def.get_solid(self.solid_handle)
 
+    @public
     @property
     def op(self) -> Node:
```

Comment:
not a public API - I think `node_name` is fine, but we should discuss if we think `Node` should be exported top level.